website/templates/home.html

```diff
@@ -19,24 +19,11 @@
 {% block title %}View campaigns{% endblock %}
 
 {% block content %}
-<!-- Hero image -->
-<div class="emblem-unpadded-grid mdc-layout-grid">
-    <div class="mdc-layout-grid__inner">
-        <div class="emblem-hero-image mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
-        </div>
-    </div>
-</div>
 
-<!-- Hero subtitle -->
 <div class="mdc-layout-grid">
     <div class="mdc-layout-grid__inner">
-        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
-            Discover a giving campaign for your contribution!
-        </div>
-        <div class="emblem-align-right mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
-            <!-- $1,234.00 contributed TODO: enhance API to support real value -->
```

Comment:
I assume omitting the "total contributed so far" amount is appropriate.

Reply:
This was removed to make the homepage resemble some to the new design. This is also why the hero image was removed.